### ISSUE TITLE
Segmentation fault (core dumped) 

### ISSUE THREAD

**Issue by healthonrails** [healthonrails/annolid#47]
Got keys from plugin meta data ("vnc")
QFactoryLoader::QFactoryLoader() looking at "/home/anaconda3/plugins/platforms/libqxcb.so"
Found metadata in lib /home/anaconda3/plugins/platforms/libqxcb.so, metadata=
{
    "IID": "org.qt-project.Qt.QPA.QPlatformIntegrationFactoryInterface.5.3",
    "MetaData": {
        "Keys": [
            "xcb"
        ]
    },
    "className": "QXcbIntegrationPlugin",
    "debug": false,
    "version": 329991
}

Got keys from plugin meta data ("xcb")
QFactoryLoader::QFactoryLoader() checking directory path "/home/anaconda3/bin/platforms" ...
loaded library "/home/anaconda3/plugins/platforms/libqxcb.so"
Segmentation fault (core dumped)

OS

Distributor ID:	Ubuntu
Description:	Ubuntu 20.04.5 LTS
Release:	20.04
Codename:	focal

**Comment by healthonrails** [comment]
Here is one of the worked solutions. 
```
conda create -n annolid python==3.7 -c conda-forge
conda install pyqt -c conda-forge
conda install pyqtwebengine -c conda-forge
conda install opencv -c conda-forge
cd annolid
pip install -e .
```

**Comment by healthonrails** [comment]
# on Ubuntu 20.04, the recommended worked solution
cd /path/to/annolid
conda env create -f environment.yml

**Comment by healthonrails** [comment]
Got keys from plugin meta data ("xcb")
QFactoryLoader::QFactoryLoader() checking directory path "/anaconda3/envs/annolid-test/bin/platforms" ...
loaded library "/anaconda3/envs/annolid-test/lib/python3.11/site-packages/PyQt5/Qt5/plugins/platforms/libqxcb.so"
Segmentation fault (core dumped)
(annolid-test):~/Downloads/annolid$ gdb --args python3 annolid/gui/app.py 

(gdb) r
Starting program: /anaconda3/envs/annolid-test/bin/python3 annolid/gui/app.py
[Thread debugging using libthread_db enabled]
Using host libthread_db library "/lib/x86_64-linux-gnu/libthread_db.so.1".
Got keys from plugin meta data ("xcb")
QFactoryLoader::QFactoryLoader() checking directory path "/anaconda3/envs/annolid-test/bin/platforms" ...
loaded library "/anaconda3/envs/annolid-test/lib/python3.11/site-packages/PyQt5/Qt5/plugins/platforms/libqxcb.so"

Thread 1 "pt_main_thread" received signal SIGSEGV, Segmentation fault.
0x00007ffec42a83e6 in ?? () from /anaconda3/envs/annolid-test/lib/python3.11/site-packages/decord/../decord.libs/libxcb-77222338.so.1.1.0
(gdb) 

Solution is : 
 ** pip install decord==0.4.0

**Comment by healthonrails** [comment]
The issue was resolved by employing decord version 0.4.0.
